meinberlin/apps/kiezradar/models.py

```diff
@@ -188,10 +190,13 @@ def get_search_profiles_for_project(project: Project) -> QuerySet[SearchProfile]
         & Q(disabled=False)
     )
     search_term = project.name
-    if connection.vendor == "postresql":
+    if connection.vendor == "postgresql":
         # django has some postgresql-only search tools which are much better
+        query = "|".join(search_term.split(" "))
```

Comment:
Isn't this the same as just passing `search_term` to `SearchQuery` and not passing `raw`? 

> If search_type is 'plain', which is the default, the terms are treated as separate keywords.
https://docs.djangoproject.com/en/5.1/ref/contrib/postgres/search/#searchquery

Reply:
yes but then it's an & and not |, so all separate keywords have to match, not just one. This would probably make sense when searching from a searchprofile for a matching project, the problem here is that we are basically reverse searching by using the project.title (etc.) to match a search profile. Or am I confusing something? :sweat_smile:

Reply:
Ahh, I think I understand. Yea, all good then!